notebooks/network_analysis_sec.ipynb

```diff
@@ -0,0 +1,523 @@
+      "Component 'Carrier' has 37 entries\n",
+      "Component 'GlobalConstraint' has 2 entries\n",
+      "Component 'Line' has 17 entries\n",
+      "Component 'LineType' has 34 entries\n",
+      "Component 'TransformerType' has 14 entries\n",
+      "Component 'Link' has 809 entries\n",
+      "Component 'Load' has 461 entries\n",
+      "Component 'Generator' has 136 entries\n",
+      "Component 'StorageUnit' has 6 entries\n",
+      "Component 'Store' has 197 entries\n"
+     ]
+    }
+   ],
+   "source": [
+    "for c in n.iterate_components(list(n.components.keys())[2:]):\n",
+    "    print(\"Component '{}' has {} entries\".format(c.name,len(c.df)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "List the snapshots of the PyPSA network"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DatetimeIndex(['2013-01-01 00:00:00', '2013-01-01 03:00:00',\n",
+      "               '2013-01-01 06:00:00', '2013-01-01 09:00:00',\n",
+      "               '2013-01-01 12:00:00', '2013-01-01 15:00:00',\n",
+      "               '2013-01-01 18:00:00', '2013-01-01 21:00:00',\n",
+      "               '2013-01-02 00:00:00', '2013-01-02 03:00:00',\n",
+      "               ...\n",
+      "               '2013-12-30 18:00:00', '2013-12-30 21:00:00',\n",
+      "               '2013-12-31 00:00:00', '2013-12-31 03:00:00',\n",
+      "               '2013-12-31 06:00:00', '2013-12-31 09:00:00',\n",
+      "               '2013-12-31 12:00:00', '2013-12-31 15:00:00',\n",
+      "               '2013-12-31 18:00:00', '2013-12-31 21:00:00'],\n",
+      "              dtype='datetime64[ns]', name='snapshot', length=2920, freq=None)\n",
+      "Time steps: 2920\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(n.snapshots)\n",
+    "print(f\"Time steps: \" + str(len(n.snapshots)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Analyse energy system"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Transport Sector \n",
+    "The demand of EVs, ICE (internal combustion engine) and FCEV (fuell cell)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "      carrier                   bus_carrier\n",
+       "Load  land transport EV         Li ion         -470.260030\n",
+       "      land transport fuel cell  H2             -235.130015\n",
+       "      land transport oil        oil           -5878.250369\n",
+       "dtype: float64"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "load_stats = n.statistics.energy_balance(comps=[\"Load\"], aggregate_time=\"mean\")\n",
```

Comment:
You use `aggregate_time=mean`, which, at least I assume, gives you the average hourly load. I think it makes more sense to go for `aggregate_time=sum` which provides an annual energy value (in MWh). Do this for all sectors.